config/environment.js

```diff
@@ -29,30 +29,31 @@ module.exports = function(environment) {
         }]
       ]
     },
-
-    web3ProviderUrl: "https://parity.kosmos.org:8545",
-    ethereumChain: "testnet",
+    contractMetadata: {},
     ipfs: {
       host: 'localhost',
       port: '5001',
       protocol: 'http'
     }
   };
+  if (process.env.KREDITS_CONTRACT_ADDR) {
```

Comment:
I see. what is the best option to set local dev configuration options? (like for example here the contract addresses?)